package.json

```diff
@@ -76,6 +76,7 @@
     "tsconfig-paths": "^3.14.1",
     "typescript": "^4.6.3",
     "vite": "^2.9.6",
+    "vite-jsconfig-paths": "^2.0.1",
```

Comment:
We'll keep using this until `vite-tsconfig-paths` is updated to use latest version of `tsconfig-paths` (which will support `jsconfig.json` files (see https://github.com/dividab/tsconfig-paths/pull/199)) once it's released

Reply:
Created https://github.com/aleclarson/vite-tsconfig-paths/issues/51 to track this in `vite-tsconfig-paths`

Reply:
https://github.com/aleclarson/vite-tsconfig-paths/pull/57 is now merged and released in [v3.5.0](https://github.com/aleclarson/vite-tsconfig-paths/releases/tag/v3.5.0), so we can remove `vite-jsconfig-paths` 🎉